guide_restli_client.md

```diff
@@ -286,6 +287,37 @@ a method to set each of them, of the form:
     public <BuilderType> <assocKeyName>Key(<AssocKeyType> value);
 ```
 
+### BATCH FINDER Request Builder
+
+In Rest.li < 1.24.4, the generated BATCH_FINDER request builder for a
+resource is named `<Resource>BatchFindBy<BatchFinderName>Builder`, while in
+Rest.li >= 1.24.4 it is named
+`<Resource>BatchFindBy<BatchFinderName>RequestBuilder`. Both builders support the
+full interface of the built-in `BatchFindRequestBuilder`.
+
+If the resource class is a child resource, the generated builder will
+include a type-safe path-key binding method for each of the resource's
+ancestors (recursively following parent resources). Each binding method
+is declared as:
+```java
+    public <BuilderType> <pathKeyName>Key(<KeyType> key);
+```
+
+The generated builder will contain a method to set each of the batchFinder's
+query parameters, of the form:
+```java
+    public <BuilderType> <paramName>Param(<ParamType> value);
+```
+
+The value **must** be non-null. For the batch query parameter, it also uses the form above
+like the other regular parameters.
+
+If the batchFinder specifies `AssocKey` parameters, the builder will contain
+a method to set each of them, of the form:
+```java
+    public <BuilderType> <assocKeyName>Key(<AssocKeyType> value);
+```
+
```

Comment:
Same here, we should add a link to the resource method doc if the reader wants more details.